app/src/main/java/comcmput301f16t01/github/carrier/Requests/Request.java

```diff
@@ -150,8 +157,10 @@ public String getId() {
 
     @Override
     public String toString() {
-        String requestAsString = "Request From: " + rider.getUsername() + "\n";
-        requestAsString += "Description: " + description;
+        String requestAsString = "Request From: " + rider.getUsername() + "\n" +
+                "Description: " + description + "\n" +
+                "Price: " + getFare() + "\n" +
```

Comment:
Do FareCalculator fc = new FareCalculator(); fc.toString(getFare()); to get a properly formatted fare (right now it is the fare * 100)